glambie/processing/main.py

```diff
@@ -0,0 +1,71 @@
+import logging
+from glambie.monitoring.logging import setup_logging
+from glambie.config.config_classes import GlambieRunConfig
+from glambie.data.data_catalogue import DataCatalogue
+from glambie.processing.process_region import run_one_region, combine_within_one_region
+from glambie.processing.path_handling import OutputPathHandler, get_output_path_handler_with_timestamped_subfolder
+
+
+log = logging.getLogger(__name__)
+
+
+def run_glambie_assessment(glambie_run_config: GlambieRunConfig,
+                           log_level: str = "INFO",
+                           verbose: bool = True):
+
+    if not log.hasHandlers():
+        setup_logging(log_level=log_level)
+    log.info('Starting GlaMBIE algorithm: log_level=%s, verbose=%s', log_level, verbose)
+
+    if verbose:
+        output_path_handler = get_output_path_handler_with_timestamped_subfolder(glambie_run_config.result_base_path)
+        log.info('Output directory set to %s', output_path_handler.base_path)
+    else:
+        output_path_handler = None
+
+    data_catalogue_original = load_catalogue_and_data(glambie_run_config.catalogue_path)
+    results_catalogue_combined_per_region, _ = run_regional_results(
+        glambie_run_config, data_catalogue_original, output_path_handler=output_path_handler)
+    run_global_results(results_catalogue_combined_per_region)
+
+
+def run_regional_results(glambie_run_config: GlambieRunConfig,
```

Comment:
Think this and the other functions in this file (except run_glambie_assessment) can me renamed with an underscore at the front (i.e. run_regional_results -> _run_regional_results) to indicate they are only used locally!

Reply:
This applies to functions in process_region.py as well (other than run_one_region())

Reply:
sure, made them "private"